README.md

```diff
@@ -649,7 +649,7 @@ maintaining the Node.js project.
 * [marsonya](https://github.com/marsonya) -
   **Akhil Marsonya** <<akhil.marsonya27@gmail.com>> (he/him)
 * [Mesteery](https://github.com/Mesteery) -
-  **Mestery** <<mestery@pm.me>>
+  **Mestery** <<mestery@protonmail.com>> (he/him)
```

Comment:
Can you add a `.mailmap` entry for this too so that your entry here matches your entry in `AUTHORS` and so that you won't show up in `AUTHORS` twice?

It should look something like this and be placed in the right place in `.mailmap` alphabetically:

```
Mestery <mestery@protonmail.com> <mestery@pm.me>
```

After you add the entry, you can test it by running `tools/update-authors.js`. That will update the AUTHORS file and automatically change your pm.me entry to protonmail.com. Then add that version of `AUTHORS` to this PR> (It may also add new entries to the end of the AUTHORS file. You can delete those or leave them as part of this PR.)

All told, three files need to be updated: `README.md`, `.mailmap`, and (via the update script) `AUTHORS`.